ios-sdk.podspec

```diff
@@ -5,15 +5,12 @@ Pod::Spec.new do |s|
 
   s.summary           = 'Mapzen iOS SDK'
   s.description       = 'Mapzen iOS SDK'
-  s.homepage          = 'https://mapzen.com/products/ios/'
+  s.homepage          = 'https://mapzen.com/projects/mobile/'
```

Comment:
This url is likely to change again in the near future but this is the best one to use for now. Alternatively you could use https://mapzen.com/documentation/ios/.